src/bundle/Resources/config/universal_discovery_widget.yml

```diff
@@ -37,6 +37,7 @@ system:
                     multiple: false
                 richtext_embed_image:
                     multiple: false
+                    allowed_content_types: ['image']
```

Comment:
As far as I know, this should be copied from the `content_on_the_fly.allowed_content_types` if it's not set by this subscriber https://github.com/ezsystems/ezplatform-admin-ui/blob/1.5/src/lib/UniversalDiscovery/Event/Subscriber/DecoupleContentOnTheFlyAllowedConfiguration.php#L38. 

It was added exactly for that case - cotf allowed content type is set by the global allowed is not.
So I would suggest investigating why it's not set by the subscriber.

Summon @Nattfarinn for confirmation.

Reply:
Talked with @Nattfarinn and seems like it was intended in 2.5 to get rid of this subscriber.